stdlib/universe/universe.flux

```diff
@@ -8,21 +14,229 @@ import "strings"
 import "regexp"
 import "experimental/table"
 
-// now is a function option whose default behaviour is to return the current system time
+// now is a function option that, by default, returns the current system time.
+//
+// #### now() vs system.time()
+// `now()` returns the current system time (UTC). `now()` is cached at runtime,
+// so all executions of `now()` in a Flux script return the same time value.
+// `system.time()` returns the system time (UTC) at which `system.time()` is executed.
+// Each instance of `system.time()` in a Flux script returns a unique value.
+//
+// ## Examples
+//
+// ### Use the current UTC time as a query boundary
+// ```no_run
+// data
+//     |> range(start: -10h, stop: now)
+// ```
+//
+// ### Define a custom now time
```

Comment:
Might want to say above (where we say the value is cached) that `now` can be overwritten.